data_management/services/dlu_filesystem.py

```
@@ -105,16 +105,20 @@ def split_path(self, path: str, preserve_path: bool = False):
+        try:
+            package_path = self.dlu_data_directory + "/" + self.dlu_package_dir_prefix + package_id
+            if os.stat(package_path).st_uid != user_id or os.stat(package_path).st_gid != int(os.environ['dlu_group']):
+                os.chown(package_path, user_id, int(os.environ['dlu_group']))
+                for file in files:
+                    os.chown(package_path + "/" + file.name, user_id, int(os.environ['dlu_group']))
+            for root, dirs, _ in os.walk(package_path):
+                for dir in dirs:
+                    subdir_path = os.path.join(root, dir)
+                    if os.stat(subdir_path).st_uid != user_id or os.stat(subdir_path).st_gid != int(os.environ['dlu_group']):
+                        os.chown(subdir_path, user_id, int(os.environ['dlu_group']))
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)